sites/en/intro-to-rails/getting_started.step

```diff
@@ -41,15 +41,69 @@ steps do
   end
 
   step do
-    console "rails new suggestotron"
-    message "`rails new` creates a new Rails project with the name you give."
-    message "In this case we told it to create a new project called `suggestotron`. We'll go into detail on what it created shortly."
-    message "This will print a lot of stuff to the screen and can take a while to finish."
-  end
+    message "Create the new Rails app:"
+
+    option_half "Without Docker" do
+      console "rails new suggestotron"
+      message "`rails new` creates a new Rails project with the name you give."
+      message "In this case we told it to create a new project called `suggestotron`. We'll go into detail on what it created shortly."
+      message "This will print a lot of stuff to the screen and can take a while to finish."
+      console "cd suggestotron"
+      message "`cd suggestotron` makes suggestotron our current directory."
+    end
 
-  step do
-    console "cd suggestotron"
-    message "`cd suggestotron` makes suggestotron our current directory."
+    option_half "With Docker" do
+      console "mkdir suggestotron"
+      console "cd suggestotron"
+
+      message "Copy this into the file `Gemfile`:"
+
+      source_code :ruby, <<-CONTENTS
+				source "https://rubygems.org"
+				gem "rails", "5.1.4"
+      CONTENTS
+
+      console "touch Gemfile.lock"
+
+      message "Copy this into the file `Dockerfile`:"
+
+      source_code :text, <<-CONTENTS
+				FROM ruby:2.5.0-slim
+
+				RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs postgresql-client libsqlite3-dev build-essential
+
+				RUN mkdir /app
+				WORKDIR /app
+				COPY Gemfile /app/Gemfile
+				COPY Gemfile.lock /app/Gemfile.lock
+				RUN bundle install
+				COPY . /app
+      CONTENTS
+
+			message "Copy this into the file `docker-compose.yml`:"
+
+			source_code :text, <<-CONTENTS
+        version: '3'
+        services:
+          web:
+            build: .
+            tty: true
+            stdin_open: true
```

Comment:
See https://github.com/pdarden/railsbridgenyc_docker/blob/master/docker-compose.yml.  We need an `environment`and `depends_on` configs so the rails app can connect to the db and students can access the rails up on their browser.